src/main/java/tetris/Controller.java

```diff
@@ -73,6 +77,25 @@ public void handleKeyEvent(KeyEvent event) {
         redraw();
     }
 
+    public void changingMusic() {
+        System.out.println(score.getScore());
```

Comment:
yes, this was just for testing when the music changed, so I could listen if it actually changed(forgot to remove it afterwards)